code/src/components/NewMessageForm.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+export const NewMessageForm = ({ newMessage, onNewMessageChange, onSubmit }) => {
+
+  return (
+    <div className='newThoughtsContainer'>
+      <form onSubmit={onSubmit} className='form'>
+        <label htmlFor='newMessage' className='newMessage'>
+          What's making you happy right now?
+        </label>
+        <textarea
+          id='newMessage'
+          type='text'
+          minLength='5'
+          maxLength='140'
+          required
```

Comment:
Instead of just NOT post the message under 5 letters, maybe in the future, you can add a popup that tells that I need to write more than 4 letters to post 😎